shape.lic

```diff
@@ -129,11 +129,18 @@ class Shape
     get_item('shaping book')
     echo('*** You will need to upgrade to a journeyman or master book before 176 ranks! ***') if DRSkill.getrank('Engineering') == 175
     turn_to("page #{find_recipe(@chapter, @recipe_name)}")
-    bput('study my book', 'Roundtime')
+    bput('study my book', 'Roundtime', 'Arrow shaft')
     stow_item('book')
     get_item('drawknife')
-    get_item("#{@material} lumber")
-    shape('scrape my lumber with my drawknife')
+    if @material == 'shafts'
+      get_item("#{@material}")
```

Comment:
You don't need string interpolation here

Reply:
get_item("#{@material}") ==> get_item(@material) ?

Reply:
Changed to get_item(@material).